docs/quick-start.txt

```diff
@@ -0,0 +1,71 @@
+.. _laravel-quick-start:
+
+===========
+Quick Start
+===========
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, odm
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+This guide shows you how to add {+odm-long+} to a new Laravel web application,
+connect to a MongoDB cluster hosted on MongoDB Atlas, and how to perform
+read and write operations on the data.
+
+.. tip::
+
+   If you prefer to set up your development environment in GitHub Codespaces
+   or Docker, see the linked code repository in the
+   `How to Build a Laravel + MongoDB Back End Service <https://www.mongodb.com/developer/languages/php/laravel-mongodb-tutorial/>`__
+   MongoDB Developer Center tutorial.
+
+   You can learn how to set up a local Laravel development environment
+   and perform CRUD operations by taking the
+   :mdbu-course:`Getting Started with Laravel and MongoDB </getting-started-with-laravel-and-mongodb>`
+   MongoDB University Learning Byte.
+
+   If you prefer to connect to MongoDB by using the PHP Library driver without
+   Laravel, see `Connecting to MongoDB <https://www.mongodb.com/docs/php-library/current/tutorial/connecting/>`__
+   in the PHP Library documentation.
+
+{+odm-short+} extends the Laravel Eloquent and Query Builder syntax to
+store and retrieve data from MongoDB.
+
+MongoDB Atlas is a fully managed cloud database service that hosts your
+MongoDB deployments. You can create your own free (no credit card
+required) MongoDB Atlas deployment by following the steps in this guide.
```

Comment:
Thanks for mentioning this. The specific wording was provided by Product a few years ago. Since we haven't received any guidance to change it, I will leave it for now.